src/main/java/org/snomed/cdsservice/model/CDSTriggerType.java

```diff
@@ -0,0 +1,5 @@
+package org.snomed.cdsservice.model;
+
+public enum CDSTriggerType {
```

Comment:
This may not be needed once we have two classes for the CDSTriggers.

Reply:
removed this file